ElevatorSim/src/elevatorsim/scheduler/Scheduler.java

```diff
@@ -129,4 +156,12 @@ public void stopRunning() {
 	public SchedulerState getSchedulerState() { 
 		return state;
 	}
-}
+	
+	public ArrayList<ElevatorRequest> getStoredRequests(){
+		return this.storedRequests;
```

Comment:
I don't like the idea of letting a foreign object have direct access to this. I'd rather them have an unmodifiable copy.

Reply:
Returns copy instead now